SharpPcap/CaptureDeviceExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace SharpPcap
+{
+    public static class CaptureDeviceExtensions
```

Comment:
how would a user know that this class exists? I don't see it being used anywhere yet. should we show it in an example? we could put this inside of ICaptureDevice, PcapDevice etc so that open variant shows up there.

Reply:
It's an extension methods class, you don't actually need to know/use the class name.
See https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/extension-methods

Reply:
Ahh perfect, I haven't used those much.